ironfish-rust/src/test_util.rs

```diff
@@ -7,13 +7,13 @@ use super::{
     witness::{Witness, WitnessNode},
     MerkleNoteHash,
 };
+use bls12_381::Scalar;
 use ironfish_zkp::constants::TREE_DEPTH;
 use rand::{thread_rng, Rng};
 
 /// Given a note, construct a Witness with a valid root_hash and authentication
 /// path placing that note at a random location in a Merkle tree.
-#[cfg(test)]
```

Comment:
Gating these here is kind of unnecessary if we gate the module definition in lib.rs (see above)